py/server/deephaven/table_factory.py

```diff
@@ -373,14 +373,14 @@ def function_generated_table(table_generator: Callable[[], Table],
         refresh_interval_ms (int): Interval (in milliseconds) at which the 'table_generator' function is rerun.
         exec_ctx (ExecutionContext) -> Table): A custom execution context. If 'None', the current
             execution context is used. If there is no current execution context, a ValueError is raised.
+        **kwargs (Dictionary): Optional keyword arguments to pass to table_generator.
```

Comment:
is `Dictionary` a valid type? Since `**kwargs` is so Python idiomatic, I don't think we need to provide type info for it.

Reply:
I think this would be reasonable.